seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumRun.scala

```diff
@@ -0,0 +1,171 @@
+package org.scalajs.jsenv.selenium
+
+import org.openqa.selenium._
+
+import org.scalajs.io._
+import org.scalajs.jsenv._
+
+import scala.annotation.tailrec
+import scala.collection.JavaConverters._
+import scala.concurrent.{Future, Promise}
+import scala.util.control.NonFatal
+
+import java.util.concurrent.ConcurrentLinkedQueue
+
+private[selenium] final class SeleniumRun(
+    driver: WebDriver with JavascriptExecutor,
+    config: SeleniumJSEnv.Config,
+    streams: OutputStreams.Streams,
+    materializer: FileMaterializer,
+    onMessage: Option[String => Unit]) extends JSComRun {
```

Comment:
I don't like that all runs extend `JSComRun`, even non-com runs. For better or worse, Scala has unconstrained `isInstanceOf` and pattern matching, so we cannot completely rule out some code somewhere deciding to pattern-match a `JSRun` to test whether it is a `JSComRun`.

I would prefer a subclass `SeleniumComRun` that extends `SeleniumRun` and defines the `send()` method.